apiserver/Makefile

```diff
@@ -68,7 +68,7 @@ build-apiserver: build-hail-base hailjwt
 build-hail-jupyter: build-hail-base jupyter_notebook_config.py
 	-docker pull $(REPO)/hail-base:latest
 	-docker pull $(REPO)/hail-jupyter:latest
-	docker build -t hail-jupyter -f Dockerfile.hail-jupyter . --cache-from hail-jupyter,$(REPO)/hail-jupyter:latest,hail-base
+	docker build -t hail-jupyter -f Dockerfile.hail-jupyter --cache-from hail-jupyter,$(REPO)/hail-jupyter:latest,hail-base ..
```

Comment:
What's the reason for this change? I thought `.` meant build from the current directory. So now you're building from the directory above?

Reply:
Yes.  I'm building up one level so I can include hailjwt, batch and pipeline that also live up one level.  The path (. or ..) is called the docker context path.  For security reasons, docker only lets docker files access things inside the context, so I can't build in . and refer to ../batch in the docker file.

Make sense?